src/main/java/com/esri/samples/featurelayers/list_releated_features/ListRelatedFeaturesSample.java

```diff
@@ -0,0 +1,167 @@
+package com.esri.samples.featurelayers.list_releated_features;
```

Comment:
Where is the license header?  Notice some samples source don't have it, we need to add the header.

Reply:
@doneill I've fixed it in this sample. I will look into fixing the other samples missing it.